### DISCUSSION TITLE
Create language/notable-design-decisions page

### DISCUSSION THREAD

**Discussion by AlexDaniel** [perl6/doc#4190] [Wanted]
Another item originating from #728.

When I started using Perl 6, I really enjoyed stuff like this: https://perl6advent.wordpress.com/2014/12/24/day-24-seeing-wrong-right/. I think it helped me see why some things are this way and not another. Also, who knows, maybe this page could give people more incentive to be open-minded to stuff like sigils.

**Comment by JJ** [comment]
Not sure about this one. Design decisions are probably implementation-dependent, and thus should go to rakudo. Will this help Perl 6 users in some way? Not sure, but probably not. 
Then there's the problem on how to document those decisions. Unless whoever took the decision does it, it's probably impossible for the average perl6/doc contributor to find that out. So if you don't feel strongly about this, I'm for closing it.

**Comment by AlexDaniel** [comment]
> Design decisions are probably implementation-dependent, and thus should go to rakudo.

I am talking about language design decisions, and that can only fit language docs.

> Will this help Perl 6 users in some way?

I think understanding why-s can help a lot when learning a language.

> Unless whoever took the decision does it, it's probably impossible for the average perl6/doc contributor to find that out

It is still possible to talk about pros and cons, and then ping one of the Larrys to check if it's correct and if there's anything to add.

**Comment by JJ** [comment]
Then we'll have to do two (big) things here:
* [ ] *Identify* which are the notable design decisions we want to talk about
* [ ] Document them
